.circleci/config.yml

```diff
@@ -146,7 +146,10 @@ jobs:
     resource_class: large
     steps:
       - checkout
-      - run: make tools
+      - run: |
+          make tools
+          # restore any go.{mod,sum} files that were updated indirectly
+          git reset --hard HEAD
```

Comment:
@alvin-huang @rboyer I think you mentioned this recently. Is this the best fix for CI in general?

Reply:
This is fine for now. That target only `go get`s a bunch of tools. However if the target ever changes and we do want something to persist in the future we can't just `reset --hard`. I wouldn't worry about it now.